XamlControlsGallery/App.xaml.cs

```diff
@@ -71,8 +71,11 @@ public void EnableSound(bool withSpatial = false)
             return (TEnum)Enum.Parse(typeof(TEnum), text);
         }
 
-        private void App_Resuming(object sender, object e)
+        private async void App_Resuming(object sender, object e)
         {
+            // We are being resumed, so lets restore our state!
+            await SuspensionManager.RestoreAsync();
```

Comment:
you are catching `SuspensionManagerException` in the other place but not here, can it not fail here?